rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -251,6 +254,43 @@ def _create_networks(self, network_create_args):
         return lswitches
 
 
+    @atomic.action_timer("ovn_network.of_check_port")
+    def _of_check_ports(self, lports, sandboxes, port_bind_args):
```

Comment:
If I understand correctly, the purpose is to evaluate the end-to-end time spent in port-binding and flow installation in HVs, right? Yes, checking "port state up" is insufficient. But there are problems of checking flows on HVs:
1. It is not accurate. The periodical poll have gaps.
2. We need to check all *related* HVs (sandboxes), instead of only the ones doing port-binding. And because of this, this approach doesn't scale, because checking flows on so many sandboxes and collect the data would take time.

Because of this, we need a notification mechanism to accurately evaluate when all the HVs have finished processing the port-binding. It is handled by [1]. I believe it is a better approach, but I haven't worked on it yet.

[1] https://github.com/openvswitch/ovn-scale-test/issues/97